sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -47,322 +44,259 @@
+        <artifactId>jackson-module-afterburner</artifactId>
+        <version>2.12.2</version>
+      </dependency>
       <dependency>
         <groupId>com.google.code.gson</groupId>
         <artifactId>gson</artifactId>
         <version>2.8.6</version>
       </dependency>
-
       <dependency>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>msal4j</artifactId>
-        <version>1.10.1</version>
+        <version>1.10.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>msal4j-persistence-extension</artifactId>
         <version>1.1.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>qpid-proton-j-extensions</artifactId>
-        <version>1.2.3</version>
-      </dependency>
-
-      <dependency>
-        <groupId>com.nimbusds</groupId>
-        <artifactId>oauth2-oidc-sdk</artifactId>
-        <version>9.4</version>
+        <version>1.2.4</version>
       </dependency>
-
       <dependency>
         <groupId>com.squareup.okhttp3</groupId>
         <artifactId>okhttp</artifactId>
         <version>4.8.1</version>
       </dependency>
-
       <dependency>
         <groupId>io.dropwizard.metrics</groupId>
         <artifactId>metrics-core</artifactId>
-        <version>4.1.19</version>
+        <version>4.1.0</version>
       </dependency>
-
       <dependency>
         <groupId>io.micrometer</groupId>
         <artifactId>micrometer-core</artifactId>
-        <version>1.6.6</version>
+        <version>1.7.0</version>
       </dependency>
-
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty-bom</artifactId>
-        <version>4.1.63.Final</version>
+        <version>4.1.65.Final</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-
-      <dependency>
-        <groupId>io.netty</groupId>
-        <artifactId>netty-tcnative</artifactId>
-        <version>2.0.38.Final</version>
-      </dependency>
-
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty-tcnative-boringssl-static</artifactId>
-        <version>2.0.38.Final</version>
+        <version>2.0.39.Final</version>
       </dependency>
-
       <dependency>
         <groupId>io.projectreactor</groupId>
         <artifactId>reactor-bom</artifactId>
-        <version>2020.0.6</version>
+        <version>2020.0.7</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-
       <dependency>
         <groupId>net.java.dev.jna</groupId>
         <artifactId>jna-platform</artifactId>
         <version>5.6.0</version>
       </dependency>
-
       <dependency>
         <groupId>org.apache.qpid</groupId>
         <artifactId>proton-j</artifactId>
-        <version>0.33.4</version>
+        <version>0.33.8</version>
+      </dependency>
+      <dependency>
```

Comment:
Let's exclude this dependency

Reply:
Why is that?

Reply:
`azure-cosmos` takes a compile-scope dependency on this. So, this should be included.